etc/docker/enterprise-gateway/bootstrap-enterprise-gateway.sh

```diff
@@ -11,17 +11,15 @@ then
 	echo "usage: docker run <docker-opts> <docker-image> <command>"
 	echo ""
 	echo "where <command> is:"
-	echo "    --jovyan ... Invokes Enterprise Gateway directly.  Useful for daemon behavior."
+	echo "    --elyra ... Invokes Enterprise Gateway directly.  Useful for daemon behavior."
```

Comment:
assuming this is the only option, and it's not shared with the other docker, do we need the `--elyra` option here? If it's just the default, I would say remove it...

Reply:
I'm incline to keep the command placeholder - renamed to `--gateway` - but if no command is provided act as if `--gateway` was provided rather than today's `--help` default.  This still allows us to specific a specific CMD in the k8s yaml and docker launch scripts, which I believe avoids ambiguity.
Are you okay with that?